tasks/deploy/lib/environment-extensions.ts

```diff
@@ -589,13 +546,10 @@ extendEnvironment((hre) => {
         }
         let { result: address } = await factory.simulate.deployAndInitialize(
             args as any,
-            {
-                gas: 5_000_000n,
-                value,
-            },
+            { gas: 10_000_000n, value },
         );
         let tx = await factory.write.deployAndInitialize(args as any, {
-            gas: 5_000_000n,
+            gas: 10_000_000n,
```

Comment:
This is a big increase. Is there a tighter bound that we can use? I'm concerned that this will result in failures if the deployer doesn't have tons of ETH (but the tx would actually still succeed) on a mainnet deployment.

Reply:
I'd be surprised if Morpho really caused this big of a jump (especially considering that it's using an external library that we should have already deployed by this point).

Reply:
IIRC it needed ~6.5mil, wdyt about updating to 7.5mil?

Reply:
Could we do the closest bound that will work? These deployments should be entirely deterministic in cost, so I don't think it pays to add padding. Let me know if I'm missing something.

Reply:
FWIW, I've been testing the deployments on a mainnet fork for sanity, so I'd see anything like this pop up there if the bound was too tight for some reason. Maybe 6.75 million would be a good compromise between exactly right and having a bit of padding?